CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## [v0.19.6](https://github.com/chef/chef-dk/tree/v0.18.30) (2016-10-17)
+[Full Changelog](https://github.com/chef/chef-dk/compare/v0.18.30...v0.19.6)
+
+**Implemented enhancements:**
+
+- Set the always\_update\_cookbooks flag by default. [\#988](https://github.com/chef/chef-dk/pull/988) ([coderanger](https://github.com/coderanger))
+- Updated InSpec to v1.2.0
+
+**Closed issues:**
+
+- ChefDK 0.18.30 issue with ruby 2.3.0 & kitchen-openstack driver in test kitchen [\#1035](https://github.com/chef/chef-dk/issues/1035)
```

Comment:
Oh this was the one I was referring to in my last review. Its was a user environment issue and they promptly closed it themselves. However its good you removed the other one too since it was a dupe of a chef issue. So best to completely remove this "closed Issues" section. Also, we try to organize changelog items in chef and chef-dk into "Fixed bugs" and "Implemented Enhancements".